.gitignore

```diff
@@ -1,13 +1,20 @@
 # If a change is made in here, add it to .npmignore as well unless you
 # explicitly want to include it in the published artifact
-node_modules
+/node_modules/*
+!/node_modules/@okta
+/node_modules/@okta/*
+!/node_modules/@okta/i18n
+/node_modules/@okta/i18n/*
+!/node_modules/@okta/i18n/dist
+/node_modules/@okta/i18n/dist/*
+!/node_modules/@okta/i18n/dist/properties
+/node_modules/@okta/i18n/dist/properties/*
+!/node_modules/@okta/i18n/dist/properties/country*.properties
+!/node_modules/@okta/i18n/dist/properties/login*.properties
 npm-debug.log
 test/SpecRunner.html
 target
 build2
 .grunt
 .sass-cache
 .widgetrc
-
-# These paths should be included in the published artifact, but not the repo
-dist
```

Comment:
Hmm. I did not see this folder. I can make this more specific - the problem is that "dist" matches node_modules/@okta/i18n/dist/ as well.